protocol/app/app.go

```diff
@@ -169,6 +169,12 @@ import (
 	vestmoduletypes "github.com/dydxprotocol/v4-chain/protocol/x/vest/types"
 
 	// IBC
+	// TODO: version?
+	ica "github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts"
```

Comment:
@lcwik To confirm we plan to upgrade this to `v8` in a follow-up PR, right?

Reply:
I assume @lcwik is tracking this somewhere else, will let him chime in!

Reply:
When I merge in the Cosmos 0.50 change then yes this will need to be updated to v8.

I won't be able to build the PR once rebasing without fixing this.